schedule/virt_autotest/kubevirt-tests-sle16.yaml

```diff
@@ -0,0 +1,27 @@
+name:           kubevirt-tests
+description:    >
+    Maintainer: Nan Zhang <nan.zhang@suse.com> qe-virt@suse.de
+    Run kubevirt tests on sle16
+vars:
+    MAX_JOB_TIME: 72000
+schedule:
+    - '{{barrier_setup}}'
+    - '{{bootup_and_install}}'
+    - '{{kubevirt_tests}}'
+conditional_schedule:
+    barrier_setup:
+        SERVICE:
+            rke2-server:
+                - virt_autotest/kubevirt_barriers
+    bootup_and_install:
+        RUN_TEST_ONLY:
+            0:
+                - installation/ipxe_install
+                - installation/agama_reboot
+                - virt_autotest/login_console
+    kubevirt_tests:
+        SERVICE:
+            rke2-server:
+                - virt_autotest/kubevirt_tests_server
+            rke2-agent:
+                - virt_autotest/kubevirt_tests_agent
```

Comment:
Why do you need a new yaml file ?

Reply:
Because the installation part is different from SLES15. If use one yaml for both sles15 and sles16, we have to add more judgment to distinguish different versions.